spec/new_relic_spec.rb

```diff
@@ -26,12 +38,30 @@
   end
 
 
+  context 'with correct setup' do
+    include_examples 'loads'
+  end
+
   context 'when NEW_RELIC_LICENSE_KEY is missing' do
     before do
       app_helper.comment_lines app_path.join('.env'), /NEW_RELIC_LICENSE_KEY/
     end
   
-    include_examples 'abort early', /NEW_RELIC_LICENSE_KEY is required/
+    context 'in the test environment' do
+      let(:app_env) { 'test' }
+
+      include_examples 'does not load'
```

Comment:
Mainly because last I checked, New Relic _really_ didn't like to be loaded in a test environment. But you're right though, it should be consistent; let me see if I can coax it.